Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -29,6 +29,43 @@ enum ReaderModeTheme: String {
     case light = "light"
     case dark = "dark"
     case sepia = "sepia"
+
+    init() {
+        self = ReaderModeTheme(rawValue: ThemeManager.instance.currentName.rawValue)?.preferredTheme() ?? .light
+    }
+}
+
+extension ReaderModeTheme {
+    func preferredTheme(for theme: ReaderModeTheme? = nil) -> ReaderModeTheme {
+        let theme = theme ?? self
+        let appWideTheme = ReaderModeTheme(rawValue: ThemeManager.instance.currentName.rawValue) ?? .light
+        
+        if appWideTheme.isThemeDark {
+            return .dark // app-wide dark overrides all
+        } else if appWideTheme.isThemeSystemCompatible && theme.isThemeSystemCompatible {
```

Comment:
A simpler way to write this if statement is `theme != .sepia`. The `appWideTheme` will always be system compatible and the only other condition is that the theme is not sepia.